VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -545,10 +545,10 @@ def get_wala_version_from_command(self):
     def get_dist_info(self):
         try:
             if 'FreeBSD' in platform.system():
-                release = re.sub('\-.*\Z', '', str(platform.release()))
+                release = re.sub('\\-.*$', '', str(platform.release()))
```

Comment:
can you please add documentation and give example why this is required.

Reply:
![image](https://github.com/Azure/azure-linux-extensions/assets/74142814/5f8192a1-187a-4b60-b3bf-5ce5ae3d5d15)

Reply:
Is this warning about specific python versions? Can we validate if this works for new and older python code?

Reply:
Yeah, have validated them in py2.7 and py3.8 versions as well.